content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -1,366 +1,463 @@
 ---
-title: Quick Start
-description: Quick start instructions to setup the Istio service mesh in a Kubernetes cluster.
+title: Istio Setup in Kubernetes
+description: Instructions to setup the Istio service mesh in a Kubernetes cluster.
 weight: 10
 keywords: [kubernetes]
 ---
 
-Quick start instructions to install and configure Istio in a Kubernetes cluster.
+Follow these instructions to install and configure Istio in a Kubernetes
+cluster.
 
 ## Prerequisites
 
-The following instructions require that you have access to a
-Kubernetes **1.9 or newer** cluster with [RBAC (Role-Based Access
-Control)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
-enabled. You will also need
-[`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-**1.9 or newer** installed. Version 1.10 is recommended.
+The following instructions require:
+
+* Access to a Kubernetes **1.9 or newer** cluster with [RBAC (Role-Based Access
+  Control)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+  enabled.
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) **1.9 or
+  newer** installed. Version **1.10** is recommended.
 
   > If you installed Istio 0.2.x,
   > [uninstall](https://archive.istio.io/v0.2/docs/setup/kubernetes/quick-start#uninstalling)
-  > it completely before installing the newer version (including the Istio sidecar
-  > for all Istio enabled application pods).
+  > it completely before installing the newer version. Remember to uninstall
+  > the Istio sidecar for all Istio enabled application pods too.
 
-### Minikube
+### Setup Minikube
 
-To install Istio locally, install the latest version of
-[Minikube](https://kubernetes.io/docs/setup/minikube/) (version 0.28.0 or later).
+1. To install Istio locally, install the latest version of
+   [Minikube](https://kubernetes.io/docs/setup/minikube/), version **0.28.0 or
+   later**.
 
-Select a [VM driver](https://kubernetes.io/docs/tasks/tools/install-minikube/#install-a-hypervisor)
-and substitute the below `your_vm_driver_choice` with the installed VM driver.
+1. Select a [VM
```

Comment:
this should be one line - I suspect the cause of liting problem

Reply:
Thank you @sdake but that is not one of the issues the linter flags. There are several other links that are split like that due to line length but it does not flag any of them. 
My question there is, what should take precedence? Trying to keep the line length or keeping links on a single line?